src/lighteval/logging/evaluation_tracker.py

```diff
@@ -222,11 +238,23 @@ def save(self) -> None:
                 results_dict=results_dict,
             )
 
+        if self.wandb is True:
+            self.push_to_wandb(
+                results_dict=results_dict,
+                details_datasets=details_datasets,
+            )
+
         if self.should_push_results_to_tensorboard:
             self.push_to_tensorboard(
                 results=self.metrics_logger.metric_aggregated, details=self.details_logger.compiled_details
             )
 
+    def push_to_wandb(self, results_dict: dict, details_datasets: dict) -> None:
+        self.wandb_run.log(
+            {**results_dict["results"]},
+        )
```

Comment:
nit: instead of logging the metrics as `custom|mmlu:astronomy|0/acc_norm` , it's better to `.replace(':', '/')` to get `custom|mmlu/astronomy|0/acc_norm`. 
Otherwise wandb creates tons of collapsible sections for every subset of the benchmark, instead of just one section for `custom|mmlu` with all the metrics inside.
![image](https://github.com/user-attachments/assets/0104c15c-61ab-43bc-9ffd-5a4208893a9a)